docs/install.rst

```diff
@@ -70,3 +70,16 @@ Install the `python-snappy` module
 .. code:: bash
 
     pip install python-snappy
+
+
+Optional crc32c install
```

Comment:
@jeffwidman Could you read this through, good enough?

Reply:
Looks great